lib/modules/manager/github-actions/extract.spec.ts

```diff
@@ -155,10 +155,112 @@ describe('modules/manager/github-actions/extract', () => {
+          currentValue: '1.2.3',
+          replaceString:
+            'actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # 1.2.3',
+        },
+        {
+          currentDigest: '1e204e9a9253d643386038d443f96446fa156a97',
+          currentValue: '1.2',
+          replaceString:
+            'actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # 1.2',
+        },
+        {
+          currentDigest: '1e204e9a9253d643386038d443f96446fa156a97',
+          currentValue: '1',
+          replaceString:
+            'actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # 1',
+        },
+        {
+          currentDigest: '1e204e9a9253d643386038d443f96446fa156a97',
+          currentValue: 'v1.2.3',
+          replaceString:
+            'actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # v1.2.3',
+        },
+        {
+          currentDigest: '1e204e9a9253d643386038d443f96446fa156a97',
+          currentValue: 'v1.2',
+          replaceString:
+            'actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # v1.2',
+        },
+        {
+          currentDigest: '1e204e9a9253d643386038d443f96446fa156a97',
+          currentValue: 'v1',
+          replaceString:
+            'actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # v1',
+        },
+        {
+          currentDigest: '1e204e9a9253d643386038d443f96446fa156a97',
+          currentValue: 'v2.1.0',
+          replaceString:
+            'actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # @v2.1.0',
+        },
+        {
+          currentDigest: '1e204e9a9253d643386038d443f96446fa156a97',
+          currentValue: 'v2.1.0',
+          replaceString:
+            'actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # pin @v2.1.0',
+        },
+        {
+          currentDigest: '1e204e9a9253d643386038d443f96446fa156a97',
+          currentValue: 'v2.1.0',
+          replaceString:
+            'actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # tag=v2.1.0',
+        },
+        {
+          currentDigest: '1e204e9a9253d643386038d443f96446fa156a97',
+          currentValue: 'v2.1.0',
+          replaceString:
+            'actions/checkout@1e204e9a9253d643386038d443f96446fa156a97  #   v2.1.0',
+        },
+        {
+          currentDigest: '1e204e9a9253d643386038d443f96446fa156a97',
+          currentValue: 'v2.1.0',
+          replaceString:
+            'actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 #v2.1.0',
+        },
+        {
+          currentDigest: '1e204e9a9253d643386038d443f96446fa156a97',
+          currentValue: 'v2.1.0',
+          replaceString:
+            'actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 #v2.1.0',
+        },
+        {
+          currentDigest: '1e204e',
+          currentValue: 'v2.1.0',
+          replaceString: 'actions/checkout@1e204e # v2.1.0',
+        },
+        {
+          currentValue: '01aecc#v2.1.0',
```

Comment:
is this a valid github action version?

Reply:
It's valid Docker version, and there is no quick fix for that